modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -24,6 +24,7 @@ Perform the migration only when an interruption in service is acceptable.
 * A recent backup of the etcd database is available.
 * A reboot can be triggered manually for each node.
 * The cluster is in a known good state, without any errors.
+* AWS security group rule in place to allow `udp` packets on the 6081 port for all nodes.
```

Comment:
The https://issues.redhat.com/browse/OCPBUGS-16200 is reported in the AWS. Not sure about other cloud provider cluster.